anyway/views/news_flash/api.py

```diff
@@ -281,3 +289,49 @@ def normalize_query_param(key, value: list):
 def normalize_query(params: dict):
     params_non_flat = params.to_dict(flat=False)
     return {k: normalize_query_param(k, v) for k, v in params_non_flat.items()}
+
+
+@roles_accepted(
+    BE_CONST.Roles2Names.Authenticated.value, BE_CONST.Roles2Names.Location_verification.value
+)
+def update_new_flash_qualifying(id):
```

Comment:
Hi Shai, יפה מאד!
There is one additional location field, `resolution`, which points to the type of location, e.g street, or interurban road. We use a dict, `resolution_dict`, that lists the location fields of each location type. It is defined in `anyway/parsers/__init__.py`. You need to also set the `resolution` field when updating news-flash location.

Reply:
i saw the dict resolution_dict and i am not sure what is the match type for road_segment_name only. 
in the dict is see that the param road1 is also required
"כביש בינעירוני": ["road1", "road_segment_name"]

Reply:
Hi Shai, I apologize it took me long to answer. I wasn't aware of this question. The table `RoadSegments` (`road_segments`) holds the road segments data. `road_segment_id` is a primary key, so it can be used to get the row of the segment, and the road number from it.